cloud/azure/deploytf/.nitric/modules/service/main.tf

```diff
@@ -43,17 +43,26 @@ data "azuread_client_config" "current" {}
 
 data "azurerm_client_config" "current" {}
 
+variable "role_names" {
```

Comment:
should this be under /service/variables.tf?

Reply:
I'm not entirely sure actually, so far the pattern is that all the logic is in the main.tf file and var.tf is mostly only used by CDKTF for class generation.